students/12th/sunghomoon/project_westagram/user/views.py

```diff
@@ -0,0 +1,61 @@
+import json
+from django.views import View
+from django.http  import JsonResponse
+from .models      import User
+
+class SignUp(View):
+  def post(self, request):
+    data = json.loads(request.body)
+
+    UserData = User(
+    phoneNumber = data['phoneNumber'],
+    name        = data['name'],
+    email       = data['email'],
+    password    = data['password'],
+    )
+
+    if UserData.email is False or UserData.password is False:
```

Comment:
파이썬에서는 값이 있으면 True 가 되므로 위와 같이 선언하시는 것 너무 좋습니다. 
파이썬에서는 Boolean 값을 `if`문으로 평가할 때 어차피 if True 인 것을 if True==True라고 사용하지 않는 편입니다.
그리고 False or False는 not(True and True)와 의미가 같아서 아래처럼 수정하셔도 같은 동작을 하게됩니다.
따라서 아래와 같은 코드가 좀 더 "파이썬다운" 코드라고 할 수 있습니다

한 번 이해하시면 그 뒤부터는 매우 편하게 사용하실 수 있을거에요! 
```suggestion
    if not(UserData.email and UserData.password):
```